modules/users/client/views/admin/view-user.client.view.html

```diff
@@ -8,7 +8,7 @@ <h1 ng-bind="vm.user.username"></h1>
         <a class="btn btn-primary" ui-sref="admin.user-edit({userId: vm.user._id})">
           <i class="glyphicon glyphicon-edit"></i>
         </a>
-        <a class="btn btn-primary" ng-click="vm.remove()" ng-if="vm.user._id !== vm.authentication.user._id">
+        <a class="btn btn-primary" ng-click="vm.remove()" ng-if="vm.user.username !== vm.authentication.user.username">
```

Comment:
Or maybe a better name for the view-model field would be `vm.isContextUserSelf` or `vm.isEditedUserSelf`.. "isCurrentUserSelf" seems confusing.